scene/gui/popup_menu.h

```diff
@@ -144,11 +144,13 @@ class PopupMenu : public Popup {
 	void _activate_submenu(int p_over, bool p_by_keyboard = false);
 	void _submenu_timeout();
 
+	int _select_previous();
+	int _select_next();
```

Comment:
On the hindsight these should probably be named differently, e.g.:
```suggestion
	int _focus_previous_selectable();
	int _focus_next_selectable();
```

Reply:
Good catch, will go with these names